File: SampleSetup/Output/Kotlin/SampleProject/Protocol/Model/ApiCity.kt

package SampleProject.Protocol.Model

import SampleProject.Protocol.Model.*
open class ApiCity: AbstractModelWithId() {
    var CountryId: String

    var Country: ApiCountry

    var TitleShort: String

    var TitleLong: String

    var Addresses: ArrayList<ApiAddress>

}

File: SampleSetup/Output/TypeScript/SampleProject/Protocol/Model/ApiCity.ts
import { AbstractModelWithId } from './AbstractModelWithId';
import { ApiCountry } from './ApiCountry';
import { ApiAddress } from './ApiAddress';
export class ApiCity extends AbstractModelWithId {
    CountryId: string;

    Country: ApiCountry;

    TitleShort: string;

    TitleLong: string;

    Addresses: Array<ApiAddress>;

}
